
class TestException(Exception):
    pass

class Test2Exception(Exception):
    """何もしない場合のブロック定義"""
